lgsm/functions/check_steamcmd.sh

```diff
@@ -63,6 +63,43 @@ fn_check_steamcmd(){
 	fi
 }
 
+fn_check_steamcmd_dir(){
+	# Worksround that pre-installs the correct steam directories to ensure all packages use the correct Standard
+	# https://github.com/ValveSoftware/steam-for-linux/issues/6976#issuecomment-610446347
+	if [ ! -d "${HOME}/.local/share/Steam" ]; then
+		mkdir -p "${HOME}/.local/share/Steam"
```

Comment:
Please use `${XDG_DATA_HOME}/Steam` if set, as per the XDG basedir spec: https://specifications.freedesktop.org/basedir-spec/latest/

The easiest way to do this is probably to do:

```
: "${XDG_DATA_HOME:="${HOME}/.local/share"}"
```

and then use `${XDG_DATA_HOME}/Steam` everywhere else.